# gFOF : Current fingertip offset in 1/10 millimeters.
uint16 g_fof

# gGWD : Current width between the gripper fingers in 1/10 millimeters.
uint16 g_gwd

# gSTA : Current device status
uint8 g_sta

# gWDF : Current width between the gripper fingers in 1/10 millimeters with offset
uint16 g_wdf